<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="ContinueOutOfFinally.xml" source-language="en-US" target-language="de-DE">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5b5b9c4cd4598afa718b5042856b4b3061a5ae7c2.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">b5b9c4cd4598afa718b5042856b4b3061a5ae7c2</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:Microsoft.JScript.ContinueOutOfFinally">
          <source>Represents the exception state when code execution continues out of a <ph id="ph1">&lt;see langword="finally" /&gt;</ph> block.</source>
          <target state="translated">Stellt den Ausnahmezustand dar, wenn die Codeausführung über einen <ph id="ph1">&lt;see langword="finally" /&gt;</ph>-Block hinaus fortgesetzt wird.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:Microsoft.JScript.ContinueOutOfFinally">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph> class.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph>-Klasse.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph> class.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph>-Klasse.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor(System.Int32)">
          <source>The target of the exception.</source>
          <target state="translated">Das Ziel der Ausnahme.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor(System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph> class, using the specified target.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph>-Klasse unter Verwendung des angegebenen Ziels.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor(System.String)">
          <source>The message that describes the error.</source>
          <target state="translated">Die Meldung, in der der Fehler beschrieben wird.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph> class, using the specified error message.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph>-Klasse unter Verwendung der angegebenen Fehlermeldung.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor(System.String,System.Exception)">
          <source>The message that describes the error.</source>
          <target state="translated">Die Meldung, in der der Fehler beschrieben wird.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Die Ausnahme, die die Ursache der aktuellen Ausnahme ist.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph> class, using the specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialisiert eine neue Instanz der <ph id="ph1">&lt;see cref="T:Microsoft.JScript.ContinueOutOfFinally" /&gt;</ph>-Klasse unter Verwendung der angegebenen Fehlermeldung und eines Verweises auf die interne Ausnahme, die Ursache dieser Ausnahme ist.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The serialized object data about the exception that is being thrown.</source>
          <target state="translated">Die serialisierten Objektdaten zur ausgelösten Ausnahme.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Contextual information about the source or destination.</source>
          <target state="translated">Kontextinformationen zur Quelle bzw. zum Ziel.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>Stores information about the exception in the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>.</source>
          <target state="translated">Speichert Informationen zur Ausnahme in der <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:Microsoft.JScript.ContinueOutOfFinally.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The stored information includes the <ph id="ph1">&lt;see cref="F:Microsoft.JScript.ContinueOutOfFinally.target" /&gt;</ph> value.</source>
          <target state="translated">Die gespeicherten Informationen enthalten den <ph id="ph1">&lt;see cref="F:Microsoft.JScript.ContinueOutOfFinally.target" /&gt;</ph>-Wert.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="F:Microsoft.JScript.ContinueOutOfFinally.target">
          <source>Stores the target of the exception.</source>
          <target state="translated">Speichert das Ziel der Ausnahme.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>